router/appRest.js

```diff
@@ -342,7 +447,16 @@ function sendResponseWhatsapp(response,responseBody, recipient, textContent) {
 }
 function sendChannelResponse(response, responseKey, data, responseCode) {
 	response.set('Content-Type', 'application/json')
-	if (responseCode) response.status(responseCode)
+	if (responseCode){
+		response.status(responseCode)
+		var edata = {
```

Comment:
Why can't you write single method in telemetry utils or wrapper function to do this
Something like below
**Caller method**
telemetry.telemetryLog(data, utils.getEdata(data, utils.Messages.ERROR_404)

**Util service:**
COST Messages = {
 "ERROR_404": "401 invalid request",
 ...
}

getEdata(data, message) {
var edata = {
		type: "system",
		level: "INFO",
		requestid: data.customData.requestid,
		message: message
	  }
	  return edata;
}

Reply:
Noted, I will do in next release code clean up task